tests/test_mail/test_mail.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from flask.ext.mail import Message
+from sourcemash.security import delay_security_email
+
+
+class TestMail:
+
+    def test_security_mail_enqueue(self, worker, outbox):
+        message = Message(subject="Welcome to Sourcemash!",
+                          recipients=['user@sourcemash.com'],
+                          sender="support@sourcemash.com")
+        delay_security_email(message)
+        worker.work(burst=True)
```

Comment:
Burst runs all jobs in the queue, which is necessary for testing in case we have another test that's running its own queue.

See: http://python-rq.org/docs/testing/